src/PIL/Image.py

```diff
@@ -633,19 +633,36 @@ def _repr_pretty_(self, p, cycle):
             )
         )
 
-    def _repr_png_(self):
+    def _repr_image(self, format):
         """iPython display hook support
 
+        :param format: Image format.
         :returns: png version of the image as bytes
         """
         b = io.BytesIO()
         try:
-            self.save(b, "PNG")
+            self.save(b, format)
         except Exception as e:
-            msg = "Could not save to PNG for display"
+            msg = f"Could not save to {format} for display"
             raise ValueError(msg) from e
         return b.getvalue()
 
+    def _repr_png_(self):
+        """iPython display hook support for PNG format.
+
+        :returns: png version of the image as bytes
+        """
+        return self._repr_image("PNG")
+
+    def _repr_jpg_(self):
+        """iPython display hook support for JPEG format.
+
+        :returns: jpg version of the image as bytes
+        """
+        return self._repr_image("JPEG")
+
+    _repr_jpeg_ = _repr_jpg_
```

Comment:
```suggestion
    def _repr_jpeg_(self):
        """iPython display hook support for JPEG format.

        :returns: jpg version of the image as bytes
        """
        return self._repr_image("JPEG")
```

Why is `_repr_jpg_` necessary? I don't see it mentioned in the [docs.](https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html#IPython.display.display)

Reply:
This method is required panel usage: https://panel.holoviz.org/reference/panes/JPG.html#parameters

Reply:
So `_repr_jpg` and `_repr_jpeg` are separate features, that you've grouped together in one PR because they are both related to JPEGs? One is for Panel, one is for IPython's `IPython.display.display_jpeg()`?

Could you post example code showing how you would use this?

Reply:
Right, `_repr_jpeg_` is required for  iPython `display_jpeg` and `_repr_jpg_` is required for Panel pane `JPG`.

I would like to use these as following (without the addition of the current PR, these does not work):

1. iPython display_jpeg
```
from IPython.display import display_jpeg
from PIL import Image

im=Image.open(<path to jpg image>)
display_jpeg(im)
```

2. Panel JPEG pane
```
from pathlib import Path
import panel as pn
from PIL import Image
pn.extension()

imb = Path(<path to jpg image>).read_bytes()
im3 = Image.open(BytesIO(imb))
jpg_pane = pn.pane.JPG(im3, width=500)

jpg_pane
```

Reply:
Okay, sure then, we can only keep `_repr_jpeg_`